### PR TITLE
update base image

### PR DESCRIPTION
**Pull request by patricklee2**
changes for php-builds

### CODE REVIEW

**Comment by patricklee2** [comment]
@sanchitmehta please review

File: kudu/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM oryxprod/build:20190521.9 as main
+FROM mcr.microsoft.com/oryx/build:20190617.2 as main
```
Comment (Hazhzeng) [inline kudu/Dockerfile]: Tested oryx build on this image.

Although in the oryx help said *--platform* is supported, but still running into an issue where `Unrecognized option '--platform' /opt/Kudu/Scripts/starter.sh oryx build /home/site/wwwroot -o /home/site/wwwroot --platform python --platform-version 3.6 -i /tmp/8d7019a264a79e8 -p packagedir=.python_packages/lib/python3.6/site-packages`

Comment (Hazhzeng) [reply kudu/Dockerfile]: The latest image 20190704.2 fix this issue, please check.

Comment (patricklee2) [reply kudu/Dockerfile]: what is the output of oryx --version?
Either way I'll bump the version

**Pull request merged**